### ISSUE TITLE
Unable to save current tabs in network drive folder

### ISSUE THREAD

**Issue by cgaddi** [usama8800/VSCode-Tab-Groups#26]
I have just installed version 1.4.8 of the extension and I have files open from a mapped network drive opened in VS Code. When attempting to save current tabs, it closed all of the tabs and my editor is stuck reloading this window.

![image](https://user-images.githubusercontent.com/72746813/120369624-c4763780-c2d0-11eb-91bc-5bb806648a55.png)

Thank you for your help!

**Comment by usama8800** [comment]
I just tried this on a mapped drive and it worked. Maybe you're having permission problems?

And just to be clear, we're talking about Windows mapped drives, right?

**Comment by cgaddi** [comment]
Yes it is Windows mapped drive. What permissions should I check for? I have the permissions to read and write all of the documents in the mapped drive.

**Comment by usama8800** [comment]
The groups are saved in `.vscode/settings.json` if you're not saving globally. If you see the value changing when you click save but the tabs are not opening it's something else

**Comment by cgaddi** [comment]
Here are the current settings that I have for Tab Groups:
![image](https://user-images.githubusercontent.com/72746813/121088280-57fcac00-c79a-11eb-8925-b6e8e80fb855.png)

When I save the tab group, it populates my settings.json with the following:
![image](https://user-images.githubusercontent.com/72746813/121088190-369bc000-c79a-11eb-85cc-573a0ac65faf.png)

But then it closes all of the open files and VS Code gets stuck reloading. When I reopen the app, the Tab Group was not saved.

If it were working properly, what should have been saved into my settings.json? Do you have an example of what it should have looked like?

**Comment by usama8800** [comment]
> ![image](https://user-images.githubusercontent.com/72746813/121088280-57fcac00-c79a-11eb-8925-b6e8e80fb855.png)

The groups here should be empty I think because your `Save Globally` is off.


> ![image](https://user-images.githubusercontent.com/72746813/121088817-220bf780-c79b-11eb-86f2-1c628b7ca202.png)

This setting is saved temporarily before saving the tabs and it is removed after saving successfully.

I'm not sure what would cause that. Was this working fine before 1.4.8? Is any tab layout not saving?

**Comment by cgaddi** [comment]
I just recently installed this extension so I don't have any experience with any of the previous versions, but I have tested the extension with local directories and those work perfectly! I have tried setting the Save Globally feature to on and it was still behaving the same way.
